src/log/log.c

```diff
@@ -59,11 +60,11 @@ __log_checksum_match(WT_ITEM *buf, uint32_t reclen)
 }
 
 /*
- * __wt_log_get_files --
+ * __log_get_files --
```

Comment:
Are these log function name changes necessary? They don't seem relevant to the PR and could cause conflicts when we merge back into dev.

Reply:
Yes we changed the scope of these functions to support the log copy stage. Effectively I'm almost reverting that commit. But it has other useful bits so I opted not to revert.